### ISSUE TITLE
Reusing existing ALB for API endpoint

### ISSUE THREAD

**Issue by mumoshu** [kubernetes-incubator/kube-aws#648]
Extracted from https://github.com/kubernetes-incubator/kube-aws/issues/608#issuecomment-300951796
Anyone is interested (to work) on it after #608 is sorted out?

**Comment by mumoshu** [comment]
Closing this as wontfix - ALB by definition won't add any value over ELB as an API endpoint load balancer! See https://github.com/kubernetes-incubator/kube-aws/issues/608#issuecomment-312750742
